terraform/service_e2e_runner.tf

```diff
@@ -190,7 +190,7 @@ resource "google_cloud_run_service_iam_member" "e2e-runner-invoker" {
 resource "google_cloud_scheduler_job" "e2e-default-workflow" {
   name             = "e2e-default-workflow"
   region           = var.cloudscheduler_location
-  schedule         = "0,10,20,30,40,50 * * * *"
+  schedule         = "0,10,20,30,40,50,55 * * * *"
```

Comment:
*/5 ?

Reply:
the idea was that they both run at the time at 0 and 55 and run on 5 minute offsets the rest of the time.